ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -741,11 +760,56 @@ const ControlledTierEraModule = (props: {
   ) : null;
 };
 
-const DARHeader = () => (
-  <FlexColumn style={styles.headerFlexColumn}>
-    <Header style={styles.headerRW}>Researcher Workbench</Header>
-    <Header style={styles.headerDAR}>Data Access Requirements</Header>
-  </FlexColumn>
+// the header outside the Fadebox
+const OuterHeader = (props: { pageMode: DARPageMode }) =>
+  props.pageMode === DARPageMode.INITIAL_REGISTRATION && (
+    <FlexColumn style={styles.initialRegistrationOuterHeader}>
+      <Header style={styles.initialRegistrationHeaderRW}>
+        Researcher Workbench
+      </Header>
+      <Header style={styles.initialRegistrationHeaderDAR}>
+        Data Access Requirements
+      </Header>
+    </FlexColumn>
+  );
+
+// the header inside the Fadebox
+const InnerHeader = (props: { pageMode: DARPageMode }) =>
+  props.pageMode === DARPageMode.INITIAL_REGISTRATION ? (
+    <div
+      data-test-id='initial-registration-header'
+      style={styles.pleaseComplete}
+    >
+      Please complete the necessary steps to gain access to the <AoU />{' '}
+      datasets.
+    </div>
+  ) : (
+    <FlexColumn>
+      <div
+        data-test-id='annual-renewal-header'
+        style={styles.renewalHeaderYearly}
```

Comment:
Thoughts about : renewalHeaderYearly or yearlyRenewalHeader?

Reply:
would it be more readable , if we create a new const for annual-renewal-header and have the innerheader as 

const InnerHeader = (props: { pageMode: DARPageMode }) =>
  props.pageMode === DARPageMode.INITIAL_REGISTRATION ? (
......) : <AnnualRenewalHeader>

Reply:
ok sure

Reply:
I want to start the name with `renewalHeader` to show that it's in that set of styles.  "Yearly" is the first text here